demos/demo-react-ts/src/components/screens/CallEndedScreen.tsx

```diff
@@ -38,6 +42,7 @@ function CallEndedScreen({
         hs_call_body: notes,
         hs_call_duration: callDuration.toString(),
         hs_call_status: callStatus || "COMPLETED",
+        hs_call_recording_url: isCallRecorded ? RECORDING_URL : null,
```

Comment:
Consider tracking the recording url itself

Reply:
by tracking you mean, on the sdk support side? Dint get your comment.

Reply:
Apologies, right now you are storing (tracking) a boolean `isCallRecorded` in the state. I was simply saying you could store the URL as a `string|null` instead of boolean. Just a suggestion!

Reply:
oh ok. should be fine as i have kept that url as constant in the file from where its being used.